README.md

```diff
@@ -1 +1,28 @@
-# actions
\ No newline at end of file
+## Fastlane GitHub Actions
+
+The Fastlane GitHub Actions provide a set of GitHub Actions to make maintaining projects easier.
+
+## Actions
+
+- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)
+
+   Adds a comment to include _fastlane_ environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).
+   
+- ✔️ [@github-actions/communicate-on-pull-request-merged](communicate-on-pull-request-merged)
+
+   Adds a comment and a label to a pull request when it is merged. Read more [here](communicate-on-pull-request-merged).
+
+## Versioning 
+
+All the actions are released in one batch. We do not support semantic versioning (yet). Reference a `latest` branch in your workflow:
+
+```yaml
+...
+  - uses: fastlane/github-actions/fastlane-env-reminder@latest
```

Comment:
Maybe add the second action here somehow as well? COnfused me why this one was singled out.

Reply:
![image](https://user-images.githubusercontent.com/10795657/68889184-b5468d80-071c-11ea-97bb-047880e636a3.png)

To put the usage in a context, this section is about how to use the action in a workflow. Do you have any ideas how we could improve the description to avoid the confusion you get? 🤔

Reply:
Nope, not really - I think this is fine.